### PR TITLE
fixed database exception in proxyimpl

### PR DESCRIPTION
**Pull request by francescolops**

### CODE REVIEW

File: climate-monitoring-client/src/main/java/climatemonitoring/ProxyImpl.java [hunk 1]
```diff
@@ -14,6 +14,8 @@
 import java.io.ObjectOutputStream;
 import java.net.Socket;
 
+import javax.xml.crypto.Data;
+
```
Comment (adellafrattina) [inline climate-monitoring-client/src/main/java/climatemonitoring/ProxyImpl.java]: What is this is import??? You should remove it

**Review by adellafrattina — COMMENTED** [review]
Should be good. Fix the import and I'll approve it

**Review by adellafrattina — APPROVED** [review]
Ok, looks good!

File: climate-monitoring-client/src/main/java/climatemonitoring/ProxyImpl.java (hunk 1 above)
Comment (ccapiferri) [reply climate-monitoring-client/src/main/java/climatemonitoring/ProxyImpl.java]: lol

**Pull request merged**